moon/vehicles/rover.py

```diff
@@ -237,16 +240,17 @@ def on_joint_effort(self, data):
                             effort = [e, -e, e, -e]
 
 
-                # steer against slope proportionately to the steepness of the slope
-                fl -= self.roll
-                fr -= self.roll
-                rl -= self.roll
-                rr -= self.roll
+                if self.drive_camera_angle == 0:
```

Comment:
is angle float? If yes then I would avoid `==`

Reply:
no, it's degrees * 100 (ie float 90.0 degrees is int 9000)